src/tests/end-to-end/tests/details-view/headings.test.ts

```diff
@@ -14,8 +14,9 @@ describe('Details View -> Assessment -> Headings', () => {
     let headingsPage: DetailsViewPage;
 
     beforeAll(async () => {
-        browser = await launchBrowser({ suppressFirstTimeDialog: true });
+        browser = await launchBrowser({ suppressFirstTimeDialog: true, addLocalhostToPermissions: true });
         targetPage = await browser.newTargetPage();
+        await browser.newPopupPage(targetPage); // Required for the details view to register as having permissions/being open
```

Comment:
Did I forget to delete this after changing how we launch the browser (i.e. after adding the `addLocalhostToPermissions` flag)

Reply:
No, I added these afterwards; these are required *in addition to* the `addLocalhostToPermissions` flag (the flag ensures that Chrome gives us permission to inject, this line ensures that our extension understands that it has those permissions)